web/src/pages/admin/users/TableUsers.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState, useEffect } from 'react';
+import { NavLink } from 'react-router-dom';
+import { ReactComponent as AddBtn } from '../../../images/add_circle.svg';
+
+export default function TableUsers({ data }) {
+  const [currentPage, setCurrentPage] = useState(1);
+  const usersPerPage = 8;
+  const pageNumbers = [];
+  const numberOfPages = Math.ceil(data.length / usersPerPage);
+
+  useEffect(() => {
+    setCurrentPage(1);
+  }, [data]);
+
+  for (let i = 1; i <= numberOfPages; i++) {
+    pageNumbers.push(i);
+  }
+
+  const paginateNext = () => {
+    setCurrentPage((currentPage) => currentPage + 1);
+  };
+
+  const paginatePrevious = () => {
+    setCurrentPage((currentPage) => currentPage - 1);
+  };
+
+  const cpfMask = (value) => {
+    return value
+      .replace(/\D/g, '')
+      .replace(/(\d{3})(\d)/, '$1.$2')
+      .replace(/(\d{3})(\d)/, '$1.$2')
+      .replace(/(\d{3})(\d{1,2})/, '$1-$2')
+      .replace(/(-\d{2})\d+?$/, '$1');
+  };
```

Comment:
What about extracting this logic to a shared file? It can be reused in other components